src/Console/ConfigurationResolver.php

```diff
@@ -702,16 +703,106 @@ private function validateRules(array $rules): void
         );
 
         if (\count($unknownFixers)) {
+            
+            $renamedRules = [
+            	'blank_line_before_return' => [
+            		'new_name' => 'blank_line_before_statement',
+            		'note' => ['statements' => ['return']]
+            	],
+                'final_static_access' =>
+            	[
+            		'new_name' => 'self_static_accessor',
+            		'note' => []
+            	],
+                'hash_to_slash_comment' =>
+            	[
+            		'new_name' => 'single_line_comment_style',
+            		'note' => ['comment_types' => ['hash']]    
+            	],
+                'lowercase_constants' =>
+            	[
+            		'new_name' => 'constant_case',
+            		'note' => ['case' => 'lower']
+            	],
+                'no_extra_consecutive_blank_lines' =>
+            	[
+            		'new_name' => 'no_extra_blank_lines',
+            		'note' => []
+            	],
+                'no_multiline_whitespace_before_semicolons' =>
+            	[
+            		'new_name' => 'multiline_whitespace_before_semicolons',
+            		'note' => []
```

Comment:
```suggestion
            		'note' => null
```

Reply:
Done in https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/5887/commits/9b72cf2dfa691f691db2da5329712d3b5449b27e